packages/mui-base/src/FocusTrap/FocusTrap.tsx

```diff
@@ -261,6 +261,8 @@ function FocusTrap(props: FocusTrapProps): JSX.Element {
           tabbable = getTabbable(rootRef.current as HTMLElement);
         }
 
+        let focusableElem: HTMLElement | null = null;
```

Comment:
Could you please remove the changes related to the Focus trap from this PR? It is unrelated to the first bug and should be solved separately. This what Olivier tried to say in https://github.com/mui/material-ui/pull/37506#pullrequestreview-1462717967 and https://github.com/mui/material-ui/issues/37330#issuecomment-1576934363.

Reply:
Done @ZeeshanTamboli